spec/workers/lionpath_export_worker_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe LionpathExportWorker do
+  let(:submission) { create(:submission) }
+
+  describe '#perform' do
+    it 'finds the submission and calls Lionpath::LionpathExport' do
+      allow(Submission).to receive(:find).with(submission.id).and_return(submission)
+
+      export_instance = instance_double(Lionpath::LionpathExport)
+      allow(Lionpath::LionpathExport).to receive(:new).with(submission).and_return(export_instance)
+      allow(export_instance).to receive(:call)
+
+      described_class.new.perform(submission.id)
+
+      expect(Submission).to have_received(:find).with(submission.id)
```

Comment:
Should this be `expect(submission).to` so we're testing the specific instance?

Reply:
It's actually the class method `find` being used to store the submission variable in the LionpathExportWorker code.